loops-arrays-objects.js

```diff
@@ -48,21 +68,45 @@ const scores = [
 //  { name: 'Rianne', score: 66, grade: 'D' }
 //  ];
 // ==========================================
-
-
-
-
+console.log(scores);
+
+for (let i = 0; i < scores.length; i++) {
+    switch (true) {
+        case (scores[i].score < 60):
+            scores[i].grade = "F";
+            break;
+        case (scores[i].score < 70):
+            scores[i].grade = "D";
+            break;
+        case (scores[i].score < 80):
+            scores[i].grade = "C";
+            break;
+        case (scores[i].score < 90):
+            scores[i].grade = "B";
+            break;
+        case (scores[i].score < 100):
+            scores[i].grade = "A";
+    }
+}
+console.log(scores);
 
 // ==========================================
 // Opdracht 2
 // Schrijf een script die e-mailadressen genereert voor al onze medewerkers. Sla dit op in een nieuwe property "email" die je toevoegt aan iedere medewerker.
-
+console.log("==============Opdracht 2============================");
 const NOVIEmployees = [
     {firstName: 'Nova', lastName: 'Eeken'},
     {firstName: 'Sam', lastName: 'Barnhoorn'},
     {firstName: 'Tessa', lastName: 'Steur'},
     {firstName: 'Mark', lastName: 'Rensen'},
 ];
+console.log(NOVIEmployees);
+for (let i = 0; i < NOVIEmployees.length; i++) {
+    console.log(NOVIEmployees[i]);
```

Comment:
Wat is de reden dat je deze console.log nog hebt laten staan?   Want heeft geen toegevoegde waarde, behalve testen of iets werkt?